build.sbt

```diff
@@ -1827,6 +1872,7 @@ lazy val `project-manager` = (project in file("lib/scala/project-manager"))
   .dependsOn(pkg)
   .dependsOn(`logging-service-logback` % Runtime)
   .dependsOn(`logging-service-telemetry` % Runtime)
+  .dependsOn(`logging-service-opensearch` % Runtime)
```

Comment:
`project-manager` has a _runtime only_ dependency on `logging-service-opensearch`. That looks like a fine _loosely coupling_.